selfdrive/car/body/values.py

```diff
@@ -0,0 +1,39 @@
+from typing import Dict, List, Union
+from dataclasses import dataclass
+
+from selfdrive.car import dbc_dict
+from selfdrive.car.docs_definitions import CarInfo
+from cereal import car
+Ecu = car.CarParams.Ecu
+
+
+class CarControllerParams:
+  ANGLE_DELTA_BP = [0., 5., 15.]
+  ANGLE_DELTA_V = [5., .8, .15]     # windup limit
+  ANGLE_DELTA_VU = [5., 3.5, 0.4]   # unwind limit
+  LKAS_MAX_TORQUE = 1               # A value of 1 is easy to overpower
+  STEER_THRESHOLD = 1.0
+
+class CAR:
+  BODY = "COMMA BODY"
+
+@dataclass
+class CommaCarInfo(CarInfo):
```

Comment:
no need for the subclass here. it's just for convenience when setting defaults when we support a bunch of models for a given make